docker/mcp_server.py

```
@@ -0,0 +1,224 @@
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)